### PR TITLE
document parallel CLI import

### PR DESCRIPTION
**Pull request by mtbc**
Documents https://github.com/openmicroscopy/openmicroscopy/pull/5828 in the light of https://trello.com/c/5xiUFcMj/37-parallel-import-docs. Staged at https://ci.openmicroscopy.org/job/OMERO-DEV-merge-docs/ws/src/omero/_build/html/users/cli/import.html and https://ci.openmicroscopy.org/job/OMERO-DEV-merge-docs/ws/src/omero/_build/html/sysadmins/import-scenarios.html.

### CODE REVIEW

File: omero/sysadmins/import-scenarios.rst [hunk 1]
```diff
@@ -258,6 +258,52 @@ Disadvantages
...
 
+
+.. _parallel_import:
+
+Parallel import
+---------------
+
+Parallel import is an experimental variant of manual CLI import for
+making large-scale imports considerably faster. This feature entails
+*significant risk* and may be changed or removed at any time. If high
```
Comment (joshmoore) [inline omero/sysadmins/import-scenarios.rst]: > may be changed or removed at any time

May see extensive changes in upcoming version?

File: omero/sysadmins/import-scenarios.rst [hunk 2]
```diff
@@ -258,6 +258,52 @@ Disadvantages
...
+thread counts are specified then the import client or OMERO server may
+stop working. It is therefore imperative that new uses of parallel
```
Comment (joshmoore) [inline omero/sysadmins/import-scenarios.rst]: "stop working"

This doesn't have more risk that other parallelism and we've even taken steps to improve it, so this is likely too strong.

**Review by joshmoore — COMMENTED** [review]
Thanks, Mark. As high-level feedback, I'd probably be a bit less aggressive with the warnings. I don't think the bar on this new feature has to be higher than general parallelism against the server.

**Review by joshmoore — APPROVED** [review]
Thanks, @mtbc. That seems like a good balance between caution and panic. I'll leave you set the bar / deadline / reminder for removing the "experimental" text.

**Comment by joshmoore** [comment]
NB: currently looking into adading an import-bulk.rst similar to import-targets.rst. Perhaps for 5.4.8 we also add an import-performance.rst?

**Comment by mtbc** [comment]
That would make sense.

**Pull request merged**